docs/source/concepts/evaluation_concepts.md

```diff
@@ -0,0 +1,40 @@
+# Evaluation Concepts
```

Comment:
why arent these core concepts?

Reply:
I didn't want to pollute the Core Concepts (e.g. Distribution / Resources, etc), with API/Dev Flow Specific Concepts.